src/Sulu/Bundle/ContactBundle/Resources/js/containers/List/fieldFilterTypes/CountryFieldFilterType.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, {Fragment} from 'react';
+import {action, observable} from 'mobx';
+import {Checkbox, CheckboxGroup, Input} from 'sulu-admin-bundle/components';
+import {AbstractFieldFilterType} from 'sulu-admin-bundle/containers';
+import countryFieldFilterTypeStyles from './countryFieldFilterType.scss';
+
+class CountryFieldFilterType extends AbstractFieldFilterType<?Array<string>> {
+    static countries: {[key: string]: string} = {};
```

Comment:
do you see advantages in setting the countries to this static property instead of accessing the `Config` service? do we use the same strategy for other config values?

Reply:
Yes, that I don't have to use the `Config` service :see_no_evil: That results in less coupling, and makes it easier to test, because the `Config` service does not need to be mocked. Also, it is [what we are doing with other components as well](https://github.com/sulu/sulu/blob/a32bf97/src/Sulu/Bundle/ContactBundle/Resources/js/index.js#L40-L49).

Reply:
alright 🙂